go.mod

```diff
@@ -357,6 +357,11 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
+require (
```

Comment:
How come these ended up in a different require block ?

Reply:
Good question.

I have no idea but it seems a new "go mod" feature. I manually merged them into the first required group.